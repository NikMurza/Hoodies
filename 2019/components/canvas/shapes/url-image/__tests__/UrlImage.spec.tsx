import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import URLImage from '../UrlImage';

describe('UrlImage component', () => {
  it('render default', async () => {
    expect(toJson(shallow(<URLImage src="assets/img.svg" />))).toMatchSnapshot();
  });

  it('change props', async () => {
    const wrapper = shallow(<URLImage src="assets/img.svg" />);
    wrapper.setProps({ src: './img.svg' });
  });

  it('unmount', async () => {
    const wrapper = shallow(<URLImage src="assets/img.svg" />);
    wrapper.unmount();
    expect(wrapper.instance()).toBeNull();
  });
});
