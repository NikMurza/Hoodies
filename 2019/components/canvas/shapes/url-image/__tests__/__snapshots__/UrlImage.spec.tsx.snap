// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UrlImage component render default 1`] = `
<Image
  image={null}
/>
`;
